File: src/main/java/org/burtelov/spotifyimporter/model/musicdata/Track.java
package org.burtelov.spotifyimporter.model.musicdata;

public class Track {
    private String title;
    private String artist;
    private String album;
    private String idSpotify;

    public Track(String title, String artist) {
        this.title = title;
        this.artist = artist;
    }

    public Track(String title, String artist, String album) {
        this.title = title;
        this.artist = artist;
        this.album = album;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public void setArtist(String artist) {
        this.artist = artist;
    }

    public void setAlbum(String album) {
        this.album = album;
    }

    public void setIdSpotify(String idSpotify) {
        this.idSpotify = idSpotify;
    }

    public String getTitle() {
        return title;
    }

    public String getArtist() {
        return artist;
    }

    public String getAlbum() {
        return album;
    }

    public String getIdSpotify() {
        return idSpotify;
    }
}

File: src/main/kotlin/org/spotify/importer/dto/music/Track.kt
package org.spotify.importer.dto.music

class Track {
    var title: String
    var artist: String
    var album: String? = null
    var idSpotify: String? = null

    constructor(title: String, artist: String) {
        this.title = title
        this.artist = artist
    }

    constructor(title: String, artist: String, album: String?) {
        this.title = title
        this.artist = artist
        this.album = album
    }
}